### PR TITLE
Updated description of metallic in SpatialMaterial

### PR DESCRIPTION
**Pull request by clayjohn**
The current description is wrong. 

Thanks to Stof who pointed the issue out on discord. :)

### CODE REVIEW

File: doc/classes/SpatialMaterial.xml [hunk 1]
```diff
@@ -221,7 +221,7 @@
 			If [code]true[/code], triplanar mapping is calculated in world space rather than object local space. See also [member uv1_triplanar].
 		</member>
 		<member name="metallic" type="float" setter="set_metallic" getter="get_metallic" default="0.0">
-			The reflectivity of the object's surface. The higher the value, the more light is reflected.
+			A high value makes the material appear more like a metal. Non-metals use [member albedo] as the diffuse color and add diffuse to the specular reflection. With non-metals, the reflection appears on top of [member albedo]. Metals use [member albedo] as a multiplier to the specular reflection and set the diffuse color to black resulting in a tinted reflection. Materials work better when fully metal or fully non-metal, values between [code]0[/code] and [code]1[/code] should only be used for blending between metal and non-metal sections. To alter the amount of reflection use [member roughness].
```
Comment (akien-mga) [inline doc/classes/SpatialMaterial.xml]: There's no `albedo` member, it's either `albedo_color` or `albedo_texture`.

Comment (clayjohn) [reply doc/classes/SpatialMaterial.xml]: Ah, right. its probably better to just say "albedo" anyway because it captures both

**Comment by akien-mga** [comment]
This can't be merged as `SpatialMaterial` was renamed to `BaseMaterial3D`. I synced the changes manually in #36146. I'll also cherry-pick this PR for 3.2.1.

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.1.

**Pull request closed without merging**